plugins/base/src/main/kotlin/translators/documentables/DefaultPageCreator.kt

```diff
@@ -611,6 +630,19 @@ internal val Documentable.customTags: Map<String, SourceSetDependent<CustomTagWr
 private val Documentable.hasSeparatePage: Boolean
     get() = this !is DTypeAlias
 
+/**
+ * @see DefaultPageCreator.sortDivergentElementsDeterministically for usage
+ */
+private val divergentDocumentableComparator =
+    compareBy<Documentable, String?>(nullsLast()) { it.dri.packageName }
```

Comment:
It can be confusing that extensions from different packages are placed in the same group. Extensions from different packages have different pages, each own one, and the group title is a link that leads to the page of the first one.

It doesn't block this PR, but a consideration for the further improvement.

Reply:
Yeah, I also noticed it, but couldn't come up with how it can be fixed, other than not grouping such extensions under one block. 

Hopefully, one day we will get rid of the left column altogether as planned, but in the meantime I can create an issue for fixing this, if you think it's significant enough.

Reply:
I haven't seen such case of extensions from different packages in stdlib so far, but it can happen in kotlinx.coroutines once #2845 is implemented.